ds-css/src/styles/custom/utilities/initial-variables.scss

```diff
@@ -0,0 +1,76 @@
+@charset "utf-8";
+// GLOBAL VARIABLES
+
+// INTRODUCTION:
+
+// Everything is in alphabetical order w/in each section
+
+// ** Only global variables, nothing pertaining to a specific element/component/object
+// Note: For all customizations to non-gelobal variables, need to find inside those more specific files.
+
+// "Updates" are for customizations to Bulma for matching up with the CBP Global Variables.
+
+  ////////////////////////// **************************** /////////////////////////
+
+//   ***  Table of Contents  ***
+
+// #1 PRIMARY COLORS
+// #2 NOTIFICATION/Alert/Message COLORS
+// #3 NOTIFICATION/Alert/Message BORDER Colors
+// #4 NOTIFICATION/Alert/Message) TEXT Colors
+// #5 (DOM Element) BACKGROUND Colors
+// #6 LINK Color
+// #7 STATE Color
+
+// Beginning of Customizations
+
+// (#1) -- CBP Brand (Primary) Colors
+
+// Updates Bulma's Global Variables in their node_modules/bulma/sass/utilities/initial-variables.scss file
+$grey:                              #555555;
+$grey-dark:                         #333333;
+$grey-darker:                       #222222; 
+$grey-light:                        #aaaaaa;
+$grey-lighter:                      #eeeeee;
+$white:                             #ffffff;
+
+// Custom Variables
+$grey-base:                         #000000;
+$grey-lightest:                     #f6f6f6;
+$grey-medium:                       #808080;
+
+// (#2) CBP Brand (Notification/Alert/Message) Colors
+
+// Custom Variables
+$cbp-danger:                        #a00000;
+$cbp-default:                       #ffffff;
+$cbp-info:                          #1460aa;
+$cbp-live:                          #ffeb7a;
+$cbp-primary:                       #00416A;
+$cbp-success:                       #008000;
+$cbp-warning:                       #f09511;
+
+// (#3) CBP Brand (Notification/Alert/Message) BORDER Colors
+
+// Custom Variables
+$cbp-danger-border:                #aa0000;
+$cbp-default-border:               #adadad;
+$cbp-info-border:                  #0f4880;
+$cbp-live-border:                  #e6d261;
+$cbp-primary-border:               #0a3055;
+$cbp-success-border:               #134d13;
+$cbp-warning-border:               #4a2e1f;
+
+// (#4) CBP Brand (Notification/Alert/Message) TEXT Colors
+
+// Custom Variables
+$cbp-warning-text:                 #a44900; 
+
+
+
+// (#7) CBP Brand STATE Colors
+
+
+// Updates Bulma's Global Variables in their node_modules/bulma/sass/utilities/initial-variables.scss file
+$link-hover:                      #001b44;
```

Comment:
1. Missing "Subtle Color Palette" background color variables.
2. Still one capitalization for "$cbp-primary" on line 49.